android/build.gradle

```diff
@@ -40,11 +43,14 @@ android {
 }
 
 dependencies {
-    implementation "androidx.work:work-runtime:2.7.1"
-    implementation "androidx.concurrent:concurrent-futures:1.1.0"
-    implementation "androidx.annotation:annotation:1.2.0"
-    implementation "androidx.lifecycle:lifecycle-livedata:2.3.1"
-    implementation "androidx.core:core:1.5.0"
-    implementation "com.squareup.okhttp3:okhttp:4.9.0"
-    implementation "com.google.code.gson:gson:2.8.6"
+    def work_version = "2.9.0"
+
+    implementation "androidx.work:work-runtime:$work_version"
```

Comment:
let's put version into the dependency, it looks like it's only used in this one so no variable is necessary